spyderlib/widgets/helperwidgets.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
```

Comment:
+1 for this new file. @goanpeca will be in charge of updating `arraybuilder` once this PR is merged.